OracleWebCenterSites/README.md

```diff
@@ -0,0 +1,422 @@
+
+##### 4. Why do I get an error message as "... RCU exists already"?
+Most likely, you're not running this command for the first time. The RCU_prefix may be present already. Drop the corresponding schemas or use a different prefix.
+
+##### 5. Where can I find RCU configuration Wizard and WebCenter Sites configuration scripts?
+See [Readme.md](dockerfiles/12.2.1.3/wcs-wls-docker-install/README.md) located at `../docker-images-master/OracleWebCenterSites/dockerfiles/12.2.1.3/wcs-wls-docker-install/README.md`
+
+##### 6. How do I configure WebCenter Sites with an On-Prem Oracle Database instance? 
+Set DB_CONNECTSTRING connection string parameter as mentioned in section [Update the environment file](#1-update-the-environment-file-1).
+
+##### 7. Alternate download location for Oracle Fusion Middleware Infrastructure and Oracle Database Images? 
+Before you build an Oracle WebCenter Sites image, download the Oracle Fusion Middleware infrastructure and Oracle Database images from the [Docker Store.](https://store.docker.com/)
+
+If you download Oracle Fusion Middleware infrastructure from Docker Store, then retag using below command:
+```
+   $ docker tag store/oracle/fmw-infrastructure:12.2.1.3 oracle/fmw-infrastructure:12.2.1.3
+```
+If you download Oracle Database from Docker Store, then retag using below command:
+```
+   $ docker tag store/oracle/database-enterprise:12.2.0.1 database/enterprise:12.2.0.1
+```
+##### 8. How do I build an Oracle Database 12.2.1.x base image?
+If you want to build your own Oracle Database image, use the Docker files and scripts in the [Oracle Database](../OracleDatabase) GitHub repository.
+
+##### 9. How do I build an Oracle Fusion Middleware Infrastructure 12.2.1.x base image?
+If you want to build your own Oracle Fusion Middleware Infrastructure image, use the Docker files and scripts in the [Oracle FMW Infrastructure](../OracleFMWInfrastructure) GitHub repository.
+
+##### 10. How to fix yum.oracle.com connectivity error?
+The errors mean that the Docker engine’s host is not able to connect to external registries for update. To access external registries and build a Docker image, set up environment variables for proxy server as below:
+```
+   export http_proxy=http://www-yourcompany.com:80 
+   export https_proxy=http://www-yourcompany.com:80 
+   export HTTP_PROXY=http://www-yourcompany.com:80 
+   export HTTPS_PROXY=http://www-yourcompany.com:80 
+   export NO_PROXY=localhost,.yourcompany.com 
+```
+##### 11. How to fix error "Please specify script.work.dir to use an alternate location” while running Admin Container?
+Make sure you have granted the right permission to 'oracle' user as described in section [Mounting a Host Directory as a Data Volume](#b-mounting-host-directory-as-a-data-volume-1).
+
+##### 12. Permission denied while connecting to the Docker daemon socket?
```

Comment:
You should call out that this allows the user root access and is not recommended for security purposes. It is preferred to use `sudo` as this provides an audit trail of activity.

Reply:
Changes incorporated into latest file.